src/checkout-buttons/strategies/paypal/paypal-button-strategy.ts

```diff
@@ -29,7 +29,11 @@ export default class PaypalButtonStrategy implements CheckoutButtonStrategy {
             throw new InvalidArgumentError();
         }
 
-        return this._paypalScriptLoader.loadPaypal()
+        if (!paymentMethod) {
```

Comment:
If we are adding this check, we need to add the corresponding test to `paypal-button-strategy-spec.ts`

Reply:
Done